blog/Introducing-Femto.md

```diff
@@ -0,0 +1,184 @@
+- title: Introducing Femto
+- subtitle: Automatic npm package resolution used by Fable bindings
+
+## Introducing Femto
+
+In this blog post, we will introduce a new tool called Femto that will hopefully make the lives of Fable users a lot easier when it comes to the npm packages they depend upon. First things first: let's go through the problem context to understand what Femto is trying to solve.
+
+### Fable Packages
+As you all know, Fable packages are normal dotnet packages that are distributed to the nuget package registry. A special type of these nuget packages are Fable bindings which provide idiomatic and type-safe wrappers around native Javascript APIs. These native Javascript APIs can be either available directly in the environment where your compiled F# code is running (Web APIs in browsers, system modules in node.js), or these APIs can be those of third-party javascript libraries distributed to the npm registry and this is where things get complicated.
+
+In order to use a Fable binding for a third-party javascript library you as a user have to install *both* libraries in your project:
+ - The Fable binding from nuget
+ - The actual javascript third-party library from npm
```

Comment:
`javascript` should be written `JavaScript`

Reply:
Done